internal/binutils/binutils.go

```diff
@@ -39,6 +40,8 @@ type Binutils struct {
 	rep *binrep
 }
 
+var objdumpLLVMVerRE = regexp.MustCompile(`((?i)LLVM) ((?i)version) (\d*)\.(\d*)\.(\d*)`)
```

Comment:
Can this be simply `(?i)LLVM) version (\d*)\.(\d*)\.(\d*)`?

Reply:
Leaving the current expression as is.

Reply:
`(?i)LLVM) version (\d*)\.(\d*)\.(\d*)` does the same but is shorter.

https://play.golang.org/p/MuS-HtOiBo4

Reply:
You can drop `(?i)` for `LLVM`. I have fixed the internal issue (lowercase `llvm`).

Reply:
Great, thanks. Done.